### ISSUE TITLE
Get Videos default thumbnail_url 403's

### ISSUE THREAD

**Issue by thedist** [twitchdev/issues#822]
**Brief description**
When calling the Get Videos endpoint, any recently created video will likely not have a generated thumbnail yet and so return a template url for the default image. This default image works at certain resolutions such as 320x180 which is used on the Twitch site itself but other resolutions do not work and result in a 403 Object not found error.

**How to reproduce**
Use the Get Videos endpoint and look at the `thumbnail_url` of any video that has the default URL `https://vod-secure.twitch.tv/_404/404_processing_%{width}x%{height}.png`

Use Twitch's resolution `https://vod-secure.twitch.tv/_404/404_processing_320x%180.png` and the thumbnail will load
Use a different resolution `https://vod-secure.twitch.tv/_404/404_processing_1920x1080.png` and the request will 403 `Object not found .`

**Expected behavior**
The template URL for the default image should work at whatever resolution specified, just like actual thumbnails.

**Comment by Xemdo** [comment]
The behavior of the URL is that it only ever supported 320x180 and a few other fixed resolutions, but now only supports 320x180.
Internally we discussed updating the default URL to not include templates, but decided to keep it as-is in case we implement dynamic resizing or additional fixed sizes in the future (if we switched the template URL, potential future changes would become breaking changes).

_Important side-note: This behavior only affects the thumbnails from Get Videos and not the thumbnails from Get Streams, as Get Streams shouldn't show the 404_processing image._

We will update the [public documentation](https://dev.twitch.tv/docs/api/reference/#get-videos) to reflect this behavior.
